webcompat/static/css/development/components/markdown.css

```diff
@@ -21,9 +21,8 @@
   /* all element */
   .wc-Markdown * {
     font-size: var(--Markdown-fontSize) !important;
-    word-wrap: break-word;
-    white-space: normal;
```

Comment:
@magsout your fix is working but introducing a new issue. see the line
http://imgur.com/a/H7nw6